borg/archiver.py

```diff
@@ -1730,12 +1730,12 @@ def main():  # pragma: no cover
     except RemoteRepository.RPCError as e:
         msg = '%s\n%s' % (str(e), sysinfo())
         exit_code = EXIT_ERROR
+    except (KeyboardInterrupt, BrokenPipeError):
+        sys.stderr.close()
+        exit_code = EXIT_SUCCESS
```

Comment:
the current definition of rc codes is:
0 = success, code reached normal end, no issues
1 = warning, code reached normal end of execution, but there were some issues (like issues with single files)
2 = error, code did not reach normal end, but somehow aborted early / unexpectedly